courses/templates/course_detail.html

```diff
@@ -143,6 +145,34 @@ <h4>{{ technique.value.sub_heading }}</h4>
     {% if course.faqs %} {% include "_faqs.html" %} {% endif %}
 
 </div>
+{% if course.program and course.program.course_lineup %}
```

Comment:
@ahmed-arbisoft as some of the PR's merged earlier so I think please keep the order of sections as shown in .psd files. as an example`FAQ` section in this template would be render earlier than `courseinprogram` page and same with others.

Reply:
I would suggest we should break down this template into separate small templates so we can use it other places whenever required. may be in separate story ?  thoughts ?

Reply:
Sure, I'll extract this out into a partials template.